### PR TITLE
Fix getting the scie platform architecture  for armv7l

### PR DESCRIPTION
**Pull request by Lauszus**

### CODE REVIEW

File: jump/src/context.rs [hunk 1]
```diff
@@ -21,6 +21,22 @@ use crate::placeholders::{self, Item, Placeholder, ScieBindingEnv};
 use crate::process::{EnvVar, Process};
 use crate::{config, CurrentExe, EnvVars, Jump, Source};
 
+#[cfg(all(
+    target_os = "linux",
+    target_arch = "arm",
+    target_pointer_width = "32",
+    target_endian = "little"
+))]
+const ARCH: &str = "armv7l";
```
Comment (jsirois) [inline jump/src/context.rs:30]: This should be exported from the jump crate and re-used like EOF magic is in the package crate: https://github.com/a-scie/jump/blob/60015e988144d8b4886ca17e5bb13c02c5fcd35a/package/src/main.rs#L12-L38

File: CHANGES.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 ## 1.4.0
 
-This release adds support for Linux s390x.
+This release adds support for Linux s390x
+and fixes the `scie.platform` and `scie.platform.arch` placeholders for armv7l.
```
Comment (jsirois) [inline CHANGES.md:6]: This will need to be its own 1.4.1 release with accompanying Cargo.toml version bump. See RELEASE.md.

**Review by jsirois — COMMENTED** [review]
Thanks @Lauszus

File: CHANGES.md (hunk 2 above)
Comment (Lauszus) [reply CHANGES.md:6]: Ahh did not realized you already had made a new release. I'll fix.

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 1.4.1
```
Comment (jsirois) [inline CHANGES.md:3]: You've completed steps 3 and 4 but you missed 1 and 2: https://github.com/a-scie/jump/blob/main/RELEASE.md

Comment (Lauszus) [reply CHANGES.md:3]: I didn't know if you wanted me to do all the changes, but I'll do that!

Comment (Lauszus) [reply CHANGES.md:3]: Done.

Comment (jsirois) [reply CHANGES.md:3]: Thanks. This way I can tag & push & get a release out ~presently.

**Pull request merged**